sql/atxs/atxs.go

```diff
@@ -40,11 +41,16 @@ func decoder(fn decoderCallback) sql.Decoder {
 		stmt.ColumnBytes(0, id[:])
 		checkpointed := stmt.ColumnLen(1) == 0
 		if !checkpointed {
+			// in future, decide ATX version base on atx_blobs.version column
```

Comment:
in the future the ATX blob shouldn't be read at all by any `sql/atxs` functions besides `atxs.LoadBlob` which just returns an `types.AtxBlob` and is probably only needed for distributed validation, gossip and sync (both of which don't even decode the data).

Reply:
Yeah, you are right! :)